src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/categories.html.twig

```diff
@@ -91,3 +91,71 @@
   </div>
 </div>
 {% endblock %}
+
+{% block category_filter_row %}
+  <div class="d-inline-block dropdown dropdown-clickable product_list_category_filter">
+    <button
+      class="btn btn-outline-secondary dropdown-toggle"
+      type="button"
+      data-toggle="dropdown"
+      aria-haspopup="true"
+      aria-expanded="false"
+    >
+      {{ 'Filter by categories'|trans({}, 'Admin.Actions') }}
+      {# Display selected category if present #}
+      {%- for choice in choices -%}
+        {% if form.vars.data and choice.value == form.vars.data %}
+          ({{ choice.label }})
+        {% endif %}
+      {%- endfor -%}
+    </button>
+    <div class="dropdown-menu category-filter-menu">
+      <div class="categories-tree-actions">
+        <a href="#" class="category_tree_filter_expand">
+          <i class="material-icons">expand_more</i>
+          {{ 'Expand'|trans({}, 'Admin.Actions') }}
+        </a>
+        <a href="#" class="category_tree_filter_collapse">
+          <i class="material-icons">expand_less</i>
+          {{ 'Collapse'|trans({}, 'Admin.Actions') }}
+        </a>
+        <a href="#" class="category_tree_filter_reset">
+          <i class="material-icons">radio_button_unchecked</i>
+          {{ 'Unselect'|trans({}, 'Admin.Actions') }}
+        </a>
+      </div>
```

Comment:
This probably needs to be buttons instead of links if you don't use them as a links

Reply:
Yep nice catch